newswires/client/src/SideNav.tsx

```diff
@@ -105,6 +107,7 @@ export const SideNav = () => {
 				}
 				onClose={() => setNavIsOpen(false)}
 			>
+				<SearchBox incremental={true} />
```

Comment:
we only use the SearchBox here I think - do we need an `incremental` flag?

Reply:
good call, it doesn't seem likely that we'll reintroduce the non-incremental version any time soon so have removed the option here: https://github.com/guardian/newswires/pull/71/commits/dd688c43f2bf31039cbf2718825a68c0a160870f